### PR TITLE
[Agent] Disown the agent process.

### PR DESCRIPTION
**Pull request by digiter**
Also add `pub get` before a smoke test.

### CODE REVIEW

**Comment by digiter** [comment]
@dnfield Do you mind taking a look at the full content of the scripts? I'd like to get advices on them.

**Comment by dnfield** [comment]
They seem sane enough to me - what are you looking for in particular?

/cc @gspencergoog who's more of a bash expert than I

File: agent/bin/ci.sh [hunk 1]
```diff
@@ -36,6 +36,8 @@ pub get
 
 dart bin/agent.dart ci -c "$CONFIG_FILE" 2>&1 >> "$LOG_FILE" &
 
+disown
```
Comment (gspencergoog) [inline agent/bin/ci.sh]: I'd put this at the end of the previous line, after the "&", so it's clear what it applies to.

File: agent/bin/ci.sh [hunk 2]
```diff
@@ -36,6 +36,8 @@ pub get
 
 dart bin/agent.dart ci -c "$CONFIG_FILE" 2>&1 >> "$LOG_FILE" &
```
Comment (gspencergoog) [inline agent/bin/ci.sh]: I think this should be:
```suggestion
dart bin/agent.dart ci -c "$CONFIG_FILE" >> "$LOG_FILE" 2>&1 &
```
In order to redirect the error output that you want.

**Comment by digiter** [comment]
Nothing particular, just advices from a bash expert.

**Pull request merged**